.github/CHANGELOG.md

```diff
@@ -203,4 +204,4 @@
 1. [ECAM] Added cockpit door video - @Benjozork (Benjamin Dupont)
 1. [MISC] Standby Instrument stays ON if emergency power should be available, bug fixes - @2hwk (2Cas#1022 on discord)
 1. [CDU] Full +/- button functionality - @lhoenig (Lukas Hoenig)
-1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
\ No newline at end of file
+1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
```

Comment:
can you guys fix it? why github web can't remove that last enter line??
also ide cmd is not working also god damnn